src/dispatch/database/service.py

```diff
@@ -448,6 +448,12 @@ def common_parameters(
     }
 
 
+CommonParameters = Annotated[
+    dict[str, int | CurrentUser | DbSession | QueryStr | Json | List[str] | List[bool] | UserRoles],
```

Comment:
This should likely be a Pydantic model or TypedDict

Reply:
I'd lean on using a Pydantic model so we can do data validation.